### DISCUSSION TITLE
Automatic translation

### DISCUSSION THREAD

**Discussion by Mark-Giesen** [hedyorg/hedy#2031] [Current student projects]
We're making progress with the new translation environment. We need to do some cleanup en synchronization, but it's going to be nice. There is a, possibly, nice feature available: Automatic translation. Every time the tool encounters a new English term it can translate this using machine translation and previous translations. Would this be a nice feature for us? 
Instead of seeing the English terms until it's translated, you'll see the automatically translated, and possibly wrong, translation. Wondering what your ideas are around this.

**Comment by Felienne** [comment]
Lovely idea @Mark-Giesen and sorry for the late reply, so may things to work on. I have changed the topic to 'Large Project Plans & Ideas' because this might be a fun project for a student to pick up!

**Comment by Felienne** [comment]
Update! @LyuFangzheng might pick this up as a project!
